cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -34,48 +38,120 @@ var (
 	cssTaskList   = []models.Task{}
 	exceptionList = []Exception{}
 
-	taskDefnARN = ""
+	taskDefnARN         = ""
+	roleName            = "E2ETestRole"
+	instanceProfileName = "E2ETestInstance"
+	policyARN           = "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role"
+)
+
+const (
+	errorCode            = 1
+	numInstancesLaunched = 1
 )
 
 func init() {
 
 	cssWrapper := wrappers.NewCSSWrapper()
 	ecsWrapper := wrappers.NewECSWrapper()
+	ec2Wrapper := wrappers.NewEC2Wrapper()
+	iamWrapper := wrappers.NewIAMWrapper()
 
+	// TODO: Change these os.Exit calls to T.Errorf. Currently unable to do so because T is not initialized until the first test.
+	// (gucumber/gucumber#28)
```

Comment:
Somehow it removed my full url. Can you prepend these with `https://github.com/` so that it's the absolute url?